pkg/appendable/appendable.go

```diff
@@ -54,12 +53,14 @@ const (
 	FieldTypeArray
 	FieldTypeBoolean
 	FieldTypeNull
-	FieldTypeTrigram
+	FieldType3gram
```

Comment:
Nitpick but this would be `FieldTypeTrigram`, `FieldTypeBigram`, and `FieldTypeUnigram`.